atd-vze/src/views/Crashes/RelatedRecordsTable.js

```diff
@@ -208,10 +208,7 @@ const RelatedRecordsTable = ({
                                     }
                                     type="select"
                                   >
-                                    {/* Show a NO DATA option only when formatValue is displayed. */}
-                                    {formatValue(row, field) === "NO DATA" && (
-                                      <option value={null}>NO DATA</option>
-                                    )}
+                                    <option value={""}>NO DATA</option>
```

Comment:
always have a null option :sparkles:

Reply:
also, i was initially confused about seeing `""` here as the value prop until i traced back to `handleInputChange` and see that we handle converting to `null` there. Thanks for working inside the bounds of this code. 🙏